src/components/graph/graph.builder.js

```diff
@@ -121,6 +122,30 @@ function buildLinkProps(link, nodes, links, config, linkCallbacks, highlightedNo
         fontWeight = highlight ? config.link.highlightFontWeight : config.link.fontWeight;
     }
 
+    if (config.node) {
+        // Arrow configuration is only available for circles for now
+        switch (config.node.symbolType) {
```

Comment:
Could you move this switch-case into a separate function? Maybe something that takes as inputs the coordinates and the config (or whatever other arguments you might require). I would call it `getNormalizedNodeCoordinates` or something like that.